### PR TITLE
Update Grafana upload dashboard method

### PR DESCRIPTION
**Pull request by vmercierfr**
# Objective

Update Grafana upload dashboard method

# Why

Document all steps to update a dashboard as maintainer or end-user

# How

- Add `grafana-dashboard-update.md`
- Update references in `README.md` and `CONTRIBUTING.md`

# Release plan

- [ ] Merge this PR

### CODE REVIEW

**Comment by vmercierfr** [comment]
#sre

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -88,14 +88,6 @@ You have to commit the changes to `go.mod` and `go.sum` before submitting the pu
...
-1. Update dashboard
-
-1. Export the dashboard
-
-    Click on `Share dashboard > Export`, then select `Export for sharing externally` and click on `View JSON`.
-
-1. Open an issue with the suggestion
-
-1. The project manager will review it and submit it to Grafana.com.
+For project's maintainers, looks at [Grafana dashboard update](docs/grafana-dashboard-update.md)
```
Comment (dcupif) [inline CONTRIBUTING.md:93]:
```suggestion
For project's maintainers, look at [Grafana dashboard update](docs/grafana-dashboard-update.md)
```

File: docs/grafana-dashboard-update.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+1. Export the dashboard
+
+    Click on `Share dashboard > Export`, then select `Export for sharing externally` and click on `Save to file`.
+
+1. Remove inputs and fix the source name
+
+    We must edit the exported dashboard because we're using a dynamic data source (usually Grafana dashboards have a hard-coded data source).
+
+    ```bash
+    FILENAME=RDS instances-1701236044179.json # Replace with your exported dashboard
```
Comment (dcupif) [inline docs/grafana-dashboard-update.md:35]:
```suggestion
    FILENAME=RDS-instances-1701236044179.json # Replace with your exported dashboard
```

**Review by dcupif — APPROVED** [review]
LGTM

**Pull request merged**